t/lib/bash/collect_mysql_info.sh

```diff
@@ -163,11 +163,18 @@ cmd_ok \
 # collect_master_logs_status
 
 if [ -n "$(get_var log_bin "$p/mysql-variables")" ]; then
+   mysql_version="$(get_var version "$p/mysql-variables")"
+   source_logs_file="mysql-binary-logs"
+   source_status_file="mysql-binary-log-status"
+   if [ "$mysql_version" '<' "8.1" ]; then
```

Comment:
Is that planned for any mariadb version ? 
```
$ if [ "10.4" '<' "8.1" ]; then echo smaller; fi
smaller
```

Reply:
Good point! We do not care about MariaDB in our tests atm but we will in future versions. Could you please file a Jira ticket, so I can plan a fix for it?

Reply:
https://perconadev.atlassian.net/browse/PT-2390